tests/h/presenters/annotation_base_test.py

```diff
@@ -1,104 +1,85 @@
 import datetime
-from unittest import mock
+
+import pytest
 
 from h.presenters.annotation_base import AnnotationBasePresenter, utc_iso8601
 
 
 class TestAnnotationBasePresenter:
-    def test_constructor_args(self):
-        annotation = mock.Mock()
-        context = mock.Mock(annotation=annotation)
-
-        presenter = AnnotationBasePresenter(context)
+    def test_constructor_args(self, annotation):
+        presenter = AnnotationBasePresenter(annotation)
 
-        assert presenter.annotation_context == context
         assert presenter.annotation == annotation
 
```

Comment:
The changes here are to:

 * Use an annotation model instead of a mock
 * Parameterise the tests